<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>科学计算器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ScientificCalculator.ico</normaloff>ScientificCalculator.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
    background-color: #000000;
    border: 2px solid #333333;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 65, 73);
color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainGirdLayout">
      <item row="2" column="0">
       <layout class="QGridLayout" name="buttonGirdLayout">
        <item row="1" column="2">
         <widget class="QPushButton" name="buttonCos">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="buttonPower">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X^n</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonRadical">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>√x</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="buttonE">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>这是自然常数e</string>
          </property>
          <property name="text">
           <string>e</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="buttonAdd">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="buttonLn">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="buttonDivid">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>÷</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="buttonFactorial">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X!</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttonSquare">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X²</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="buttonMultiply">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>×</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="buttonLg">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>这是以10为底的对数</string>
          </property>
          <property name="text">
           <string>lg</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="button8">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonRightBracket">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="buttonSubtract">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="buttonPI">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>π</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="button4">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonC">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="button2">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="buttonTan">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="button0">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>这是数字0</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="button1">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="button6">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="button7">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="buttonSpot">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>这是小数点</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttonSin">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonLeftBracket">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="buttonEqual">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>这是等号</string>
          </property>
          <property name="styleSheet">
           <string notr="true"> border-radius:5px;  
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(237, 33, 58), stop:1 rgba(147, 41, 30));</string>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="button5">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="button3">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="button9">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTextBrowser" name="historyDisplay">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这是历史记录，可以在开始按钮里清除&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:10px</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="textDisplay">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>这是显示输入的表达式的地方</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:5px</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBegin">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionClearHistory"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuBegin"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(57, 57, 57);color:white;
font: 11pt &quot;微软雅黑&quot;;
</string>
   </property>
  </widget>
  <action name="actionClearHistory">
   <property name="text">
    <string>清除历史信息</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于本软件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
